index.md

```diff
@@ -85,12 +84,12 @@ There are three types of examinations/tests held in PES:
 Normal assignments of around 20 marks, conducted in the class and written in blue books. 1 assignment is conducted for every unit, and the aggregate marks are considered for the final score.
 
 !!!info 
-There are no lab exams for Physics or Chemistry. Only for C and Python.
+There are no lab exams for Physics or Chemistry. Only for C and Python. In Physics, however, there may be a 30-mark lab MCQ.
```

Comment:
Isn't the 30-marks lab MCQ a part of the ESA??

Reply:
Yeah it is, i guess. But it's based on the lab component and added to the theory final scores.